src/pages/WorkspaceSwitcherPage.tsx

```diff
@@ -0,0 +1,307 @@
+    }, [policies, getIndicatorTypeForPolicy, hasUnreadData, isOffline]);
+
+    const filteredAndSortedUserWorkspaces = useMemo(
+        () =>
+            usersWorkspaces
+                .filter((policy) => policy.text?.toLowerCase().includes(searchTerm?.toLowerCase() ?? ''))
+                .sort((policy1, policy2) => sortWorkspacesBySelected(policy1, policy2, activeWorkspaceID)),
+        [searchTerm, usersWorkspaces, activeWorkspaceID],
+    );
+
+    const usersWorkspacesSectionData = useMemo(
+        () => ({
+            data: filteredAndSortedUserWorkspaces,
+            shouldShow: true,
+            indexOffset: 0,
+        }),
+        [filteredAndSortedUserWorkspaces],
+    );
+
+    const everythingSection = useMemo(() => {
+        const option = {
+            reportID: '',
+            text: CONST.WORKSPACE_SWITCHER.NAME,
+            icons: [
+                {
+                    source: Expensicons.ExpensifyAppIcon,
+                    name: CONST.WORKSPACE_SWITCHER.NAME,
+                    type: CONST.ICON_TYPE_AVATAR,
+                },
+            ],
+            brickRoadIndicator: getIndicatorTypeForPolicy(undefined),
+            boldStyle: hasUnreadData(undefined),
+        };
+
+        return (
+            <>
+                <View style={[styles.mh4, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, styles.mb3]}>
+                    <Text
+                        style={styles.label}
+                        color={theme.textSupporting}
+                    >
+                        {translate('workspace.switcher.everythingSection')}
+                    </Text>
+                </View>
+                <View>
+                    <OptionRow
+                        option={option}
+                        onSelectRow={selectPolicy}
+                        showTitleTooltip={false}
+                        highlightSelected
+                        isSelected={!activeWorkspaceID}
+                        optionIsFocused={!activeWorkspaceID}
+                    />
+                </View>
+            </>
+        );
+    }, [activeWorkspaceID, getIndicatorTypeForPolicy, hasUnreadData, selectPolicy, styles, theme.textSupporting, translate]);
+
+    const headerMessage = filteredAndSortedUserWorkspaces.length === 0 ? translate('common.noResultsFound') : '';
+
+    const workspacesSection = useMemo(
+        () => (
+            <>
+                <View style={[styles.mh4, styles.mt6, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, ...(usersWorkspaces.length > 0 ? [styles.mb1] : [styles.mb3])]}>
+                    <View>
+                        <Text
+                            style={styles.label}
+                            color={theme.textSupporting}
+                        >
+                            {translate('common.workspaces')}
+                        </Text>
+                    </View>
+                    <PressableWithFeedback
```

Comment:
nitpick: we should have added a tooltip for `add workspace` button.

Reply:
@rushatgabhane thanks for the comment. what should the tooltip have displayed? 😄

Reply:
"Create workspace"